venues/icaps-conference.org/ICAPS/2019/Workshop/VRDIP/python/init_conference.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
```

Comment:
For convention we name the init file as 'init.py'

Reply:
I've changed the file name.  By moving the submission content anonymity up to 4, do I have to change the order number on all of the other content fields in order to get it to show up near the top?

Reply:
Ready to merge?